packages/core/src/checkout-buttons/strategies/braintree/braintree-venmo-button-strategy.ts

```diff
@@ -72,6 +89,10 @@ export default class BraintreeVenmoButtonStrategy implements CheckoutButtonStrat
         );
         const paymentMethod = state.paymentMethods.getPaymentMethodOrThrow(methodId);
         const { clientToken, initializationData } = paymentMethod;
+        const { paymentButtonStyles } = initializationData;
+        const { cartButtonStyles } = paymentButtonStyles || {};
+        const styles =
+            (braintreevenmo as BraintreePaypalButtonInitializeOptions)?.style || cartButtonStyles;
```

Comment:
Is it possible to avoid using "as" here?

Reply:
Done